Example/Emission.xcodeproj/project.pbxproj

```diff
@@ -615,6 +709,94 @@
 			};
 			name = Release;
 		};
+		51D5E9101D92814800C74594 /* Deploy */ = {
```

Comment:
Just checking - are these the build/deploy configurations that helped with the AppHub JS problem? :o